cmd/kubelet/app/server.go

```diff
@@ -597,6 +622,94 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 	return nil
 }
 
+func updateTransport(clientConfig *restclient.Config, clientCertificateManager certificate.Manager) error {
```

Comment:
looks like the dynamic kubelet config loader is the only other user of CreateAPIServerClientConfig (today)... need a follow-up to plumb the cert-rotating-enabled client to the background process responsible for monitoring config